test/watcher.js

```diff
@@ -102,6 +102,22 @@ group('chokidar is installed', function (beforeEach, test, group) {
 		logger.finish.reset();
 		logger.reset.reset();
 
+		avaFiles.reset();
+		avaFiles.defaultExcludePatterns.reset();
+		avaFiles.defaultIncludePatterns.reset();
```

Comment:
https://github.com/sindresorhus/ava/pull/713/files#r58517908 notwithstanding, it is indeed good practice to reset stub state in the `beforeEach` :smile: